src/test/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/SemanticAnalyzerScalarFunctionTest.java

```diff
@@ -104,21 +106,25 @@ public void substringWithLogFunctionCallWithUnknownFieldShouldPass() {
                 );
     }
 
+    @Ignore /** nested functions are blocked by throwing SqlFeatureNotImplementedException yet before implemented */
```

Comment:
Why ignore the test cases? Should we add the exception assert?

Reply:
That was because these cases are not valid for now since the plugin does not support these features so far, but they are remained here for future implementation. 
UTs and ITs are added in `AntlrSqlParseTreeVisitorTest.java` and `QueryAnalysis.java` instead.